lm75bd/lm75bd.c

```diff
@@ -24,11 +24,24 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
   return ERR_CODE_SUCCESS;
 }
-
+#define I2C_WRITE_BUFF_SIZE 1U
+#define I2C_READ_BUFF_SIZE 2U
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
-  return ERR_CODE_SUCCESS;
+  if (temp){
```

Comment:
Make this a guard clause, just as you diet later on, as if there are many checks the function can get very indented and hard to follow